app/src/main/java/com/nutomic/syncthingandroid/service/EventProcessor.java

```diff
@@ -93,16 +94,21 @@ public void onDone(long lastId) {
      */
     @Override
     public void onEvent(Event event) {
+        String deviceId;
+        String folderId;
+
         switch (event.type) {
             case "ConfigSaved":
                 if (mApi != null) {
                     Log.v(TAG, "Forwarding ConfigSaved event to RestApi to get the updated config.");
                     mApi.reloadConfig();
                 }
                 break;
+            case "DeviceConnected":
+                break;
```

Comment:
You are probably trying to sort it alphabetically, but I'd rather just wedge all the unhandled events at the bottom with a fallthrough down to the bottom and then break.